Directory.Build.props

```diff
@@ -95,7 +95,7 @@
     <!--CVE-2024-43485-->
     <SystemTextJsonVersion>8.0.5</SystemTextJsonVersion>
     <!--CVE-2023-29331-->
-    <SystemFormatsAsn1Version>8.0.1</SystemFormatsAsn1Version>
+    <SystemFormatsAsn1Version>9.0.8</SystemFormatsAsn1Version>
```

Comment:
@Avery-Dunn: was this necessary? In general we try to keep the dependencies minimal?

Reply:
Released an MSAL hotfix so we no longer need to update that version here.